src/Depend.sh

```diff
@@ -56,6 +56,10 @@ if (test $1 = "ASPHERE") then
   depend INTEL
 fi
 
+if (test $1 = "BPM") then
```

Comment:
This is not needed if you have a hard dependency. You need to uninstall RHEO first if you uninstall BPM or else you get an error.